sdk/appconfiguration/azure-spring-cloud-feature-management/README.md

```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
 
 #### Targeting in an Application
 
-An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample) example project.
+An example web application that uses the targeting feature filter is available in the [example project](../../spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample).
```

Comment:
we need to have complete link. Having the related reference link will cause trouble in Microsoft Doc site.

Reply:
Updated. Thank you for your check.